x-pack/plugin/sql/sql-proto/src/main/java/org/elasticsearch/xpack/sql/proto/StringUtils.java

```diff
@@ -26,7 +27,7 @@
 
     public static final String EMPTY = "";
     
-    private static final DateTimeFormatter ISO_WITH_MILLIS = new DateTimeFormatterBuilder()
+    private static final DateTimeFormatter ISO_DATE_WITH_MILLIS = new DateTimeFormatterBuilder()
```

Comment:
Doesn't the JDBC driver have access to StringUtils proto? Meaning these formatters would not be needed anymore...

Reply:
Good catch, removing and using `StringUtils`.